addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -238,10 +238,19 @@ def setUpClass(cls):
         # groups@.. will cause the creation of new mail.test.gateway
         cls.alias = cls.env['mail.alias'].create({
             'alias_name': 'groups',
+            'alias_domain': 'test.com',
             'alias_user_id': False,
             'alias_model_id': cls.test_model.id,
             'alias_contact': 'everyone'})
 
+        cls.test_model_std = cls.env['ir.model']._get('mail.test.track')
+        cls.alias_std = cls.env['mail.alias'].create({
```

Comment:
why `alias_std` ? :disguised_face: (same for `test_model_std`)

Reply:
This was because the class of this model (mail.test.track) is named MailTestStandard. I have changed test_model_std to test_model_track and alias_std to alias_valid. For the later, I was undecided to named it alias_track or alias_valid but decided to name it alias_valid to refer to the alias_name which is “valid”.